plugins/org.eclipse.xtext.builder.standalone/src/org/eclipse/xtext/builder/standalone/incremental/BuildRequest.xtend

```diff
@@ -25,6 +35,12 @@ class BuildRequest {
 	List<URI> dirtyFiles = newArrayList
 	List<URI> deletedFiles = newArrayList
 	
+	IIssueHandler issueHandler = new IIssueHandler.DefaultIssueHandler()
```

Comment:
Maybe we should rename this class. It used to be the request (data provided by the client only) before, now it gets additional data on setup (like the BuildContext) and collects even more during the build (used to be in the BuildResultCollector). Not sure whether this change is for the better.